src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1675,6 +1686,7 @@ def process_vmss_create_namespace(cmd, namespace):
     _validate_vmss_create_nsg(cmd, namespace)
     _validate_vm_vmss_accelerated_networking(cmd.cli_ctx, namespace)
     _validate_vm_vmss_create_auth(namespace, cmd)
+    _validate_trusted_launch(namespace)
```

Comment:
@sirfamelin May I ask do we need to add this warning log for flex VMSS?

Reply:
Yes